app/constants/languageMenu.js

```diff
@@ -15,5 +15,6 @@ export default {
   ru: 'русский',
   uk: 'українська мова',
   'zh-cn': '简体中文',
-  'zh-tw': '繁體中文'
+  'zh-tw': '繁體中文',
+  sv: 'Svenska'
```

Comment:
Small thing, I know, but can we keep this file in alphabetical order for easier maintenance?